cache_images/ubuntu_packaging.sh

```diff
@@ -79,13 +89,12 @@ INSTALL_PACKAGES=(\
     libnl-3-dev
     libprotobuf-c-dev
     libprotobuf-dev
+    libseccomp
     libseccomp-dev
-    libseccomp2
```

Comment:
Package name should still be `libseccomp2` and not `libseccomp`. See https://gitlab.com/rhcontainerbot/libseccomp/-/blob/debian/debian/control . `libseccomp` is just the main/source package. The generated binary/library is still libseccomp2.

Reply:
fixed